### DISCUSSION TITLE
为什么 Unilang 未将 C++ 版本设定为更高，而是使用了一个个人维护的跨平台能力无保障的类库？

### DISCUSSION THREAD

**Discussion by shirok1** [linuxdeepin/developer-center#3424] [Development Questions | 开发答疑]
[README](https://github.com/linuxdeepin/unilang/blob/master/README.zh-CN.md?plain=1#L494)

[linuxdeepin/unilang](https://github.com/linuxdeepin/unilang)

[YSLib 待定支持平台](https://frankhb.github.io/YSLib-book/Home.zh-CN.html#%E5%BE%85%E5%AE%9A%E6%94%AF%E6%8C%81%E5%B9%B3%E5%8F%B0)

是否有计划未来升级 C++ 目标版本替换 YSLib 的计划？

**Comment by BLumia** [comment]
我不是 unilang 项目的维护者所以不保证能给出可靠的答复，不过如果你问的是 YSLib 的话，YSLib 和 Unilang 是同作者的...

**Reply by shirok1** [reply]
看出来了……
